docs/src/main/paradox/common/timeouts.md

```diff
@@ -23,10 +23,6 @@ akka.http.host-connection-pool.idle-timeout
 akka.http.host-connection-pool.client.idle-timeout
 ```
 
-@@@ note
-For the client side connection pool, the idle period is counted only when the pool has no pending requests waiting.
```

Comment:
I guess this was true for the old client pool, but hasn't been the case for a while?

Reply:
Actually, it should still apply but only for the `host-connection-pool.idle-timeout` which is a different timeout than the other ones listed here. The other ones are all connection-level timeouts, while the pool timeout is for the pool. Maybe the paragraph could be broken down into two?